src/components/Person/Person.jsx

```diff
@@ -1 +1,37 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    isMarried,
+    partnerName,
+    sex,
+  } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      {age
```

Comment:
```suggestion
      {age &&
```

and remove ternary operator